building/windowsdockerbuild.cmd

```diff
@@ -0,0 +1,15 @@
+@echo off
+
+set MSBUILD="C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\MSBuild\15.0\Bin\MSBuild.exe"
+set target="Windows"
+
+if exist %MSBUILD% (
+    dotnet restore ..\src
+    %MSBUILD% msbuild.targets /l:FileLogger,Microsoft.Build.Engine;logfile=msbuild.log /target:%target%
```

Comment:
this build does not really use the docker image, does it?

Reply:
This file was only used in the docker container to build SuperDump.

I changed it in commit https://github.com/Dynatrace/superdump/pull/162/commits/2ae9708064b9c855674179e8634a44046339c134 to start the build in the docker container.